controllers/drplacementcontrol_controller.go

```diff
@@ -1633,13 +1721,43 @@ func (d *DRPCInstance) getVRGFromManifestWork(mw *ocmworkv1.ManifestWork) (*rmn.
 func (d *DRPCInstance) updateDRPCStatus() error {
 	d.log.Info("Updating DRPC status")
 
-	d.instance.Status.LastUpdateTime = metav1.Now()
+	drpc := &rmn.DRPlacementControl{}
+	drpcLookupKey := types.NamespacedName{
+		Name:      d.instance.Name,
+		Namespace: d.instance.Namespace,
+	}
+
+	err := d.reconciler.Client.Get(d.ctx, drpcLookupKey, drpc)
+	if err != nil {
+		d.log.Error(err, "failed to update status1")
+
+		return fmt.Errorf("%w", err)
+	}
+
+	drpc.Status = d.instance.Status
```

Comment:
So, why are we getting DRPC resource from the API server instead of using d.instance? (i.e. instance field from the DRPCInstance structure)

Reply:
Shyam asked the same question some time ago.  We'll remove it later.  But unit test modifies the instance and that was causing an update issue.  We'll think of a different strategy.

Reply:
ACK